jar-service/src/main/java/com/sixsq/slipstream/application/RootApplication.java

```diff
@@ -231,19 +251,13 @@ private Directory attachDownloadsDirectory() {
 		return directory;
 	}
 
-	private void attachReports(RootRouter router)
-			throws ConfigurationException, ValidationException {
+	private void attachReports(RootRouter router) throws ConfigurationException, ValidationException {
 		router.attach("/reports", new ReportRouter(getContext(), router.getApplication()));
 	}
```

Comment:
> Is / (forward slash) OK in this case in the root URI?
Yes, because we are using it directly into router.attach(...) instead of first passing it to convertToRouterRoot(...).